README.md

```diff
@@ -23,6 +23,12 @@ You can find [the documentation here](https://imitation.readthedocs.io/en/latest
 
 ## Installation
 
+### Prerequisites
+
+- Python 3.8+
+- (Optional) GLU (to render gym environments)
```

Comment:
This is a bit opaque, can we add a link? I think GLU usually refers to https://en.wikipedia.org/wiki/OpenGL_Utility_Library -- do we really depend on this, I thought it got deprecated a while back?

Looking at our Dockerfile we have:
```
    ffmpeg \
    libgl1-mesa-dev \
    libgl1-mesa-glx \
    libglew-dev \
    libosmesa6-dev \
```
Which seems like a few different GL packages plus ffmpeg

Reply:
It was what was missing on my system when I tried to run the tests initially, but I'll check again

Reply:
This is the issue I get (with a reproducible example using gym, for added context):
```bash
>>> import gym; gym.make('CartPole-v0').render()
Traceback (most recent call last):
  File "/home/i/venv/lib/python3.10/site-packages/gym/envs/classic_control/rendering.py", line 27, in <module>
    from pyglet.gl import *
  File "/home/i/venv/lib/python3.10/site-packages/pyglet/gl/__init__.py", line 95, in <module>
    from pyglet.gl.gl import *
  File "/home/i/venv/lib/python3.10/site-packages/pyglet/gl/gl.py", line 45, in <module>
    from pyglet.gl.lib import link_GL as _link_function
  File "/home/i/venv/lib/python3.10/site-packages/pyglet/gl/lib.py", line 149, in <module>
    from pyglet.gl.lib_glx import link_GL, link_GLU, link_GLX
  File "/home/i/venv/lib/python3.10/site-packages/pyglet/gl/lib_glx.py", line 46, in <module>
    glu_lib = pyglet.lib.load_library('GLU')
  File "/home/i/venv/lib/python3.10/site-packages/pyglet/lib.py", line 168, in load_library
    raise ImportError('Library "%s" not found.' % names[0])
ImportError: Library "GLU" not found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/i/venv/lib/python3.10/site-packages/gym/core.py", line 295, in render
    return self.env.render(mode, **kwargs)
  File "/home/i/venv/lib/python3.10/site-packages/gym/envs/classic_control/cartpole.py", line 179, in render
    from gym.envs.classic_control import rendering
  File "/home/i/venv/lib/python3.10/site-packages/gym/envs/classic_control/rendering.py", line 29, in <module>
    raise ImportError(
ImportError: 
    Error occurred while running `from pyglet.gl import *`
    HINT: make sure you have OpenGL installed. On Ubuntu, you can run 'apt-get install python-opengl'.
    If you're running on a server, you may need a virtual frame buffer; something like this should work:
    'xvfb-run -s "-screen 0 1400x900x24" python <your_script.py>'
```
So GLU seems to be the library they are looking for, but clearly, and solutions in https://stackoverflow.com/questions/50446867/importerror-library-glu-not-found suggest installing the GLU library.

It appears that GLU itself is not deprecated, but features it depends on are deprecated in OpenGL. (a bit more discussion here: https://stackoverflow.com/questions/41580315/are-glutess-functions-deprecated )

Clearly, the docker setup doesn't seem to require installing GLU per se, but since it is quite old, I would be inclined to see whether anything requires updating, but that would take some time which I don't want to spend fiddling with Docker right now.

I'd suggest an alternative solution to change "GLU" to "OpenGL", which is more transparent, albeit not the complete picture, but leaving it open to be made more precise in a future change.

Reply:
This is the issue I get (with a reproducible example using gym, for added context):
```bash
>>> import gym; gym.make('CartPole-v0').render()
Traceback (most recent call last):
  File ".../lib/python3.10/site-packages/gym/envs/classic_control/rendering.py", line 27, in <module>
    from pyglet.gl import *
  File ".../lib/python3.10/site-packages/pyglet/gl/__init__.py", line 95, in <module>
    from pyglet.gl.gl import *
  File ".../lib/python3.10/site-packages/pyglet/gl/gl.py", line 45, in <module>
    from pyglet.gl.lib import link_GL as _link_function
  File ".../lib/python3.10/site-packages/pyglet/gl/lib.py", line 149, in <module>
    from pyglet.gl.lib_glx import link_GL, link_GLU, link_GLX
  File ".../lib/python3.10/site-packages/pyglet/gl/lib_glx.py", line 46, in <module>
    glu_lib = pyglet.lib.load_library('GLU')
  File ".../lib/python3.10/site-packages/pyglet/lib.py", line 168, in load_library
    raise ImportError('Library "%s" not found.' % names[0])
ImportError: Library "GLU" not found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../lib/python3.10/site-packages/gym/core.py", line 295, in render
    return self.env.render(mode, **kwargs)
  File ".../lib/python3.10/site-packages/gym/envs/classic_control/cartpole.py", line 179, in render
    from gym.envs.classic_control import rendering
  File ".../lib/python3.10/site-packages/gym/envs/classic_control/rendering.py", line 29, in <module>
    raise ImportError(
ImportError: 
    Error occurred while running `from pyglet.gl import *`
    HINT: make sure you have OpenGL installed. On Ubuntu, you can run 'apt-get install python-opengl'.
    If you're running on a server, you may need a virtual frame buffer; something like this should work:
    'xvfb-run -s "-screen 0 1400x900x24" python <your_script.py>'
```
So GLU seems to be the library they are looking for, but clearly, and solutions in https://stackoverflow.com/questions/50446867/importerror-library-glu-not-found suggest installing the GLU library.

It appears that GLU itself is not deprecated, but features it depends on are deprecated in OpenGL. (a bit more discussion here: https://stackoverflow.com/questions/41580315/are-glutess-functions-deprecated )

Clearly, the docker setup doesn't seem to require installing GLU per se, but since it is quite old, I would be inclined to see whether anything requires updating, but that would take some time which I don't want to spend fiddling with Docker right now.

I'd suggest an alternative solution to change "GLU" to "OpenGL", which is more transparent, albeit not the complete picture, but leaving it open to be made more precise in a future change.

Reply:
Changing this to OpenGL seems reasonable. GLU depends on OpenGL after all, so just listing GLU on its own seems incomplete. The most helpful thing here would probably be to list what packages to install on common operating systems, but that'll be tricky to test/maintain, so keeping it vague for now is likely best.